src/index.html

```diff
@@ -4,9 +4,329 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <div class="header__content">
+        <a href="#" class="logo-link">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo-link__img"
+          >
+        </a>
+        <nav class="nav">
```

Comment:
```suggestion
        </a>
        
        <nav class="nav">
```
Add one empty line between sibling blocks inside the same parent block for both of them. Same about all the following similar cases like between `li` tags